### PR TITLE
Configs improvement

### PR DESCRIPTION
**Pull request by ikiba**
* Improves `webpack` configs to be more straightforward.
* Adds `.idea` generated by Webstorm to `.gitignore`
* Fixes `.eslintrc` bug. Improves `import/order` grouping rule.

### CODE REVIEW

File: cfg/dev.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-const { merge }            = require( 'lodash' );
-
-const baseConfig           = require( './base' );
-
-
-const devConfig = merge( {}, baseConfig, {
-    entry  : path.join( __dirname, '../src/index.js' ),
+module.exports = baseConfig( {
+    mode   : 'development',
+    inline : false,
```
Comment (conor-cafferkey-sociomantic) [inline cfg/dev.js]: We need to inline the styles for LochNess
```suggestion
    inline : true,
```

**Comment by conor-cafferkey-sociomantic** [comment]
This should be merged after https://github.com/sociomantic-tsunami/nessie-ui/issues/917

**Comment by conor-cafferkey-sociomantic** [comment]
Since merging `Milestone-Hooks!` ➡️ `master-x` we have a conflict in `cfg/dist.js`. @Sheynigami  could you look at resolving that please 🙏

**Comment by conor-cafferkey-sociomantic** [comment]
oh, wait, you’re @ikiba now? 😄

**Comment by ikiba** [comment]
yeah :smile:

**Pull request merged**